libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerProvider.kt

```diff
@@ -49,6 +62,18 @@ fun logI(msg: String, category: String? = null) {
     LoggerProvider.frontend.logI(msg, category)
 }
 
+/**
+ * @param category optional string to identify the source or category of the log message.
+ * @param lazyMsg is a lazy message to log. Isn't executed if current log level less verbose than Inline.
```

Comment:
What is "current log level less verbose than Inline"?
It also should be info, not inline. But I still don't get the meaning.

Reply:
> It also should be info, not inline 

thanks 👍
I should have waited for tomorrow instead of quickly finishing PR this evening 🤣

Reply:
> What is "current log level less verbose than Inline"?

After fixing grammar errors and typos it sounds like " The lazy message isn't executed if current log level is less verbose than Info". With less verbose log level you see less details in the logs. For example Error is less verbose than Info, if you set current log level to Error you will see less logs. It doesn't make sense to execute lazy message with verbose level Info if current log level Error, because nobody will see the message anyway. Does it make sense?

Reply:
Ah, you've fixed that already.